src/ears.cpp

```diff
@@ -36,6 +37,7 @@ npe_arg(f, dense_int, dense_long, dense_longlong)
 
 npe_begin_code()
 
+  //assert_valid_3d_tri_mesh_faces(f, "f");
```

Comment:
Why is this commented out?

Reply:
There are some tests commented out, basically because they are either from the unmerged PR, like assert_valid_tet_or_tri_mesh_faces, or they are not implemented yet, just I think they should be there so I put it, like assert_valid_3d_tri_mesh_faces.